assets/acquia_lift.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file
+ * Contains the definition of the behaviour openAcquiaLiftExperienceBuilder.
+ */
+
+(function ($, Drupal) {
+
+    'use strict';
+
+    /**
+     * Attaches the JS test behavior to to weight div.
+     */
+    Drupal.behaviors.acquiaLiftOpenExperienceBuilder = {
+        attach: function (context, settings) {
+            $('.openLiftLink').click(function() {
+                if (window.AcquiaLiftPublicApi) {
+                    window.AcquiaLiftPublicApi.activate();
+                } else {
+                    alert('Sorry, Acquia Lift could not be found');
```

Comment:
How is this case possible? Why is this not a dependency of the asset library (where you're currently adding a dependency on `core/jquery`)? See `hook_library_info_build()` for how to define dynamic libraries, that allow you to create an asset library outside of `MODULENAME.libraries.yml`, and can include e.g. an asset URL that's entered by the user.

Also, why do you have asset URLs entered by users? Why is it not an API key?